server/test_utils.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"github.com/mattermost/mattermost-plugin-confluence/server/config"
+	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/mattermost/mattermost-server/v6/plugin/plugintest"
+)
+
+const (
+	validUserID    = "iu73atknztnctef8b8ey9gm6zc"
+	validChannelID = "tgniw3kmrjd93qns11cboditme"
+)
+
+func baseMock() *plugintest.API {
+	mockAPI := &plugintest.API{}
+	config.Mattermost = mockAPI
+
+	return mockAPI
+}
+
+func getMockAPIAndPlugin() (*plugintest.API, *Plugin) {
+	mockAPI := baseMock()
+	mockAPI.On("LogError", mockAnythingOfTypeBatch("string", 13)...)
+	mockAPI.On("LogDebug", mockAnythingOfTypeBatch("string", 11)...)
+	mockAPI.On("GetBundlePath").Return("/test/testBundlePath", nil)
+
+	p := Plugin{}
+	p.SetAPI(mockAPI)
+
+	return mockAPI, &p
+}
+
+func getValidUserAndChannelID() (string, string) {
+	return validUserID, validChannelID
+}
+
+func getMockUser() *model.User {
```

Comment:
```suggestion
func getMockUser(isAdmin bool) *model.User {
```
Then, you can add the roles based on the bool variable.